src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,25 @@
+import { useParams} from 'react-router-dom'
+import { useEffect, useState } from "react"
+import { getArticleById } from './utils/api'
+
+const ArticlePage = () => {
+    const [specificArticle, setSpecificArticle] = useState({})
+    const {article_id} = useParams()
+        useEffect(() => {
+            getArticleById(article_id).then((article) => {
+                setSpecificArticle(article)
+            })
+        }, [])
+    return (
+        <>
+            <div id='specific-article-container'>
+                <h2>{specificArticle.title}</h2>
+                <h3>By {specificArticle.author}</h3>
+                <img src={specificArticle.article_img_url} />
+                <p>{specificArticle.body}</p>
```

Comment:
we should also show the number of votes, comments and the time it was created in a more human readable format - especially if we dont have this on the main articles page